packages/cli/package.json

```diff
@@ -66,7 +66,6 @@
     "@types/debug": "0.0.31",
     "@types/dotenv": "6.1.1",
     "@types/escape-html": "0.0.20",
-    "@types/express": "4.17.13",
```

Comment:
This is already [the latest types package version](https://www.npmjs.com/package/@types/express?activeTab=versions) and it still includes the [vulnerable version of `body-parser`](https://github.com/vercel/vercel/security/dependabot/5504).